spec/system/network_configuration_spec.rb

```diff
@@ -20,15 +20,15 @@
   describe 'resolving DNS entries', dns: true do
     let(:dns) { Resolv::DNS.new(nameserver: @env.dns_host) }
 
-    it 'forward looks up instance' do
+    xit 'forward looks up instance' do
```

Comment:
Instead of globally disabling these tests and changing this file, could you run your pipelines with `BAT_RSPEC_FLAGS="--tag ~dns ..."` to exclude these tests? It looks like they're already tagged. I don't think we're in a position to disable all of the DNS tests yet for our other stemcells. Other than that, I think this looks good.

Reply:
Ok, I'll revert this change.